packages/data-point/lib/core/transform.test.js

```diff
@@ -22,138 +22,94 @@ beforeAll(() => {
   });
 });
 
-test("transform - throw error in invalid id(promise)", () => {
-  return Transform.transform(dataPoint, "INVALID", TestData, {})
-    .catch(err => err)
-    .then(res => {
-      expect(res).toBeInstanceOf(Error);
-      expect(res).toMatchSnapshot();
-    });
-});
-
-describe("transform - should attach input value to accumulator", () => {
-  test("passing undefined", () => {
-    return Transform.transform(dataPoint, value => {
-      expect(value).toBe(undefined);
+describe("transform", () => {
+  test("transform - throw error in invalid id(promise)", async () => {
+    await expect(
+      Transform.transform(dataPoint, "INVALID", TestData, {})
+    ).rejects.toThrowErrorMatchingSnapshot();
+  });
+
+  describe("transform - should attach input value to accumulator", () => {
```

Comment:
this is the only change on the codebase that was "outside of scope" the tests before were too repetitive to keep, if anyone feels strong on placing it back to what it was im happy to revert